lib/vanagon/project.rb

```diff
@@ -101,6 +101,15 @@ def get_replaces
       replaces.flatten.uniq
     end
 
+    def get_service(name)
```

Comment:
On line 224, we have
```
     def get_services
       @components.map(&:service).flatten.compact
     end
```

Would it be worth it to use this method rather than adding a new one? It probably makes for cleaner code down the road if we make a new method...

Could you add documentation to this?